src/test/java/com/softserveinc/ita/vsaroz/PracticeTest.java

```diff
@@ -11,11 +11,11 @@
 
 public class PracticeTest {
 
-    private final SelenideElement productElement = $x("//a[@href='http://automationpractice.com/index.php?id_product=3&controller=product']");
-    private final SelenideElement addToCartButtonElement = $x("//button[@class='exclusive']//span");
-    private final SelenideElement closeModalButtonElement = $x("//span[@class='cross']");
-    private final SelenideElement viewShoppingCartButtonElement = $x("//a[@href='http://automationpractice.com/index.php?controller=order']");
-    private final SelenideElement itemsInCart = $x("//span[@class='ajax_cart_quantity']");
+    private final String productElement = ("//a[@href='http://automationpractice.com/index.php?id_product=3&controller=product']");
```

Comment:
after your changes it stopped being an element, currently, it's "path", delete brackets ()